modules/nw-ovn-k-anp-nbctl.adoc

```diff
@@ -0,0 +1,496 @@
+match               : "outport == @a14645450421485494999 && ((ip4.src == $a6786643370959569281)) && tcp && tcp.dst==7564"
+meter               : acl-logging
+name                : "ANP:cluster-control:Ingress:1"
+options             : {}
+priority            : 26599
+severity            : []
+tier                : 1
+
+_uuid               : 1a27d30e-3f96-4915-8ddd-ade7f22c117b
+action              : allow-related
+direction           : from-lport
+external_ids        : {direction=Egress, gress-index="3", "k8s.ovn.org/id"="default-network-controller:AdminNetworkPolicy:cluster-control:Egress:3:None", "k8s.ovn.org/name"=cluster-control, "k8s.ovn.org/owner-controller"=default-network-controller, "k8s.ovn.org/owner-type"=AdminNetworkPolicy, port-policy-protocol=None}
+label               : 0
+log                 : false
+match               : "inport == @a14645450421485494999 && ((ip4.dst == $a10622494091691694581))"
+meter               : acl-logging
+name                : "ANP:cluster-control:Egress:3"
+options             : {apply-after-lb="true"}
+priority            : 26597
+severity            : []
+tier                : 1
+
+_uuid               : b23a087f-08f8-4225-8c27-4a9a9ee0c407
+action              : allow-related
+direction           : from-lport
+external_ids        : {direction=Egress, gress-index="0", "k8s.ovn.org/id"="default-network-controller:AdminNetworkPolicy:cluster-control:Egress:0:udp", "k8s.ovn.org/name"=cluster-control, "k8s.ovn.org/owner-controller"=default-network-controller, "k8s.ovn.org/owner-type"=AdminNetworkPolicy, port-policy-protocol=udp}
+label               : 0
+log                 : false
+match               : "inport == @a14645450421485494999 && ((ip4.dst == $a13517855690389298082)) && udp && udp.dst==5353"
+meter               : acl-logging
+name                : "ANP:cluster-control:Egress:0"
+options             : {apply-after-lb="true"}
+priority            : 26600
+severity            : []
+tier                : 1
+
+_uuid               : d14ed5cf-2e06-496e-8cae-6b76d5dd5ccd
+action              : allow-related
+direction           : to-lport
+external_ids        : {direction=Ingress, gress-index="0", "k8s.ovn.org/id"="default-network-controller:AdminNetworkPolicy:cluster-control:Ingress:0:None", "k8s.ovn.org/name"=cluster-control, "k8s.ovn.org/owner-controller"=default-network-controller, "k8s.ovn.org/owner-type"=AdminNetworkPolicy, port-policy-protocol=None}
+label               : 0
+log                 : false
+match               : "outport == @a14645450421485494999 && ((ip4.src == $a14545668191619617708))"
+meter               : acl-logging
+name                : "ANP:cluster-control:Ingress:0"
+options             : {}
+priority            : 26600
+severity            : []
+tier                : 1
+
+----
+====
++
+[NOTE]
+====
+The outputs for ingress and egress show you the logic of the policy in the ACL. For example, every time a packet matches the provided `match` the `action` is taken. See ""
```

Comment:
Is there supposed to be something in the quotations after "see"?

Reply:
nope